pynestml/utils/ast_utils.py

```diff
@@ -1878,13 +1902,20 @@ def find_model_by_name(cls, model_name: str, models: Iterable[Union[ASTNeuron, A
         return None
 
     @classmethod
-    def get_convolve_function_calls(cls, ast):
+    def get_convolve_function_calls(cls, nodes: Union[ASTNode, List[ASTNode]]):
         """
         Returns all sum function calls in the handed over meta_model node or one of its children.
         :param ast: a single meta_model node.
         :type ast: ASTNode
```

Comment:
Can be removed.

Reply:
?

Reply:
I meant the type in docstring can be removed.

Reply:
Oh, of course, sorry.